.github/workflows/hydrate-universe.yml

```
@@ -0,0 +1,182 @@
+          args=()
+          [ -n "${IN_UNIVERSE}" ] && args+=("--universe=${IN_UNIVERSE}")
+          [ -n "${IN_LIMIT}" ] && args+=("--limit=${IN_LIMIT}")
+          [ "${IN_DRY_RUN}" = "true" ] && args+=("--dry-run")
+
+          # Tee so the summary can be built from the same output the log shows,
+          # rather than re-deriving counts from a second source that could disagree.
+          node scripts/hydrate-local.mjs "${args[@]}" 2>&1 | tee hydrate.log
+
+      - name: Summarize
+        if: always()
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:19:3: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)